shell/platform/fuchsia/flutter/vulkan_surface.cc

```diff
@@ -19,7 +19,24 @@ namespace flutter_runner {
 
 namespace {
 
+// Immutable format is technically limited to R8G8B8A8_SRGB but
+// R8G8B8A8_UNORM works with existing ARM drivers so we allow that
+// until we have a more reliable API for creating external Vulkan
+// images using sysmem. fxb/52835.
```

Comment:
nit: Wrap the fxb/ in a TODO() so it can be more easily found by grepping

Reply:
Done.